Formula/ruby@1.8.7.rb

```diff
@@ -0,0 +1,205 @@
+class RubyAT187 < Formula
+  desc "Powerful, clean, object-oriented scripting language"
+  homepage "https://www.ruby-lang.org/"
+  url "https://cache.ruby-lang.org/pub/ruby/1.8/ruby-1.8.7-p374.tar.bz2"
+  sha256 "b4e34703137f7bfb8761c4ea474f7438d6ccf440b3d35f39cc5e4d4e239c07e3"
+  revision 2
+
+  bottle do
+    sha256 "bc89a74b22eec03454e42a37ce12df2fd31765e3ed9f87673fb3d073f0d3db05" => :sierra
+    sha256 "85ffdf739e1f4f74979e59616933e57a3a384bfc2214277b34120295ab0a695d" => :el_capitan
+    sha256 "f40e11d81cdae7051600b72ea5511b9995420618a13b65be439f6978e634d8c5" => :yosemite
+  end
+
+  option :universal
+  option "without-suffix", "Do not suffix commands with '187' (will become keg-only)"
```

Comment:
Let's pick one: either install these as `ruby` and make them keg-only or install them with a suffix. Let's remove the option regardless. For Wikimedia it sounds like they may want `rbenv` instead.